chart/compass/values.yaml

```diff
@@ -660,9 +660,11 @@ global:
     unsecuredPort: 8083
     basicSecuredPort: 8084
     oauthSecuredPort: 8085
-    ordGlobalRegistryPort: 8086
+    ordGlobalRegistryCertPort: 8086
+    ordGlobalRegistryUnsecuredPort: 8087
```

Comment:
what is the purpose of this unsecured port, I found he's declared in the ext-svc-mock deployment's port section and as part of ext-svc-mock service but there is no virtual service to route the traffic to this server? In contrast `ordGlobalRegistryCertPort` is referred in VS